src/arg_max_min.js

```diff
@@ -1,26 +1,29 @@
 'use strict';
 
+import {cast} from './cast.js';
 import {Tensor, sizeOfShape} from './lib/tensor.js';
 import {reduceMax, reduceMin, selectValuesToReduce} from './reduce.js';
 import {squeeze} from './squeeze.js';
 
 /**
  * Get the index location of the minimum or maxmium values of all the input values along the axes.
  * @param {Tensor} input
+ * @param {Number} axis
  * @param {Function} reduceFunc
  * @param {MLArgMinMaxOptions} [options]
  * @return {Tensor}
  */
 export function argMaxMin(
     input,
+    axis,
     reduceFunc,
     {
-      axes = null,
       keepDimensions = false,
-      selectLastIndex = false,
+      outputDatatype = 'int32',
     } = {}) {
   // If axes aren't present (defaulting to null), all dimensions are reduced.
   // See https://webmachinelearning.github.io/webnn/#dom-mlargminmaxoptions-axes.
+  const axes=[axis];
```

Comment:
Please simplify these `axes` codes on line#27 and line#30~line#32, thanks.

Reply:
done